Makefile

```diff
@@ -141,17 +141,9 @@ endif
 # Name of this Python package on Pypi
 package_name := pywbemtools
 
-# Name of a specific module within the package_name.  This should be a
-# subdirectory of the package in git.
-# TODO: Future: expand so that multiple modules can reside in the package
-module_name := pywbemcli
-
-# Name of the Python package modules
-# TODO future: expand so multiple modules are allowed. Right now we have
-# one module and directly reference it in the Makefile
-pywbemcli_module_path := $(package_name)/$(module_name)
-
-pywbemcli_module_import_name := $(package_name).$(module_name)
+# Names of the commands in the package. The command names are used in
+# certain directory names.
+command1 := pywbemcli
```

Comment:
Just to make more explanable could we chage to commandpywbemcli or something rather than 1

Reply:
NO CHANGE - When searching for direct uses of 'pywbemcli' it helps if it does not show up in the variable names.